docs/howto.md

```diff
@@ -39,17 +39,23 @@ int main() {
     ozo::rows_of<std::int64_t, std::optional<std::string>> rows;
 
     // Connection info with host and port to connect to
-    auto conn_info = ozo::make_connection_info("host=... port=...");
+    // This also supports a PostgreSQL connection URI string
+    ozo::connection_info<> conn_info = ozo::make_connection_info("host=... port=...");
 
     // For _SQL literal
     using namespace ozo::literals;
+
+    std::int64_t myParam = ...;
```

Comment:
It's better to provide valid code samples.
You can put all the additional information in the comment above the variable definition.